_data/projects/tutorialdb.yml

```
@@ -0,0 +1,11 @@
+name: tutorialdb
+desc: A search engine for programming/dev tutorials.
+site: https://github.com/Bhupesh-V/tutorialdb
+tags:
+- django
+- web
+- python3
+- javascript
+upforgrabs:
+  name: help-wanted
```

We use the GitHub API to find these values, so they need to match:

```suggestion
  name: help wanted 🙌
```